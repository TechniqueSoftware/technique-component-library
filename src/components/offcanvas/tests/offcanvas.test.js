import React from 'react'
import { render, screen } from '@testing-library/react'
import Offcanvas from '..'

describe('<Offcanvas>', () => {
  it('should match snapshot', () => {
    const { asFragment } = render(<Offcanvas />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('renders', () => {
    render(
      <Offcanvas show>
        <p>offcanvas</p>
      </Offcanvas>
    )
    expect(screen.getByText('offcanvas')).toBeInTheDocument()
  })

  it('renders with title', () => {
    render(
      <Offcanvas show title='title'>
        <p>offcanvas</p>
      </Offcanvas>
    )
    expect(screen.getByText('title')).toBeInTheDocument()
  })
})
