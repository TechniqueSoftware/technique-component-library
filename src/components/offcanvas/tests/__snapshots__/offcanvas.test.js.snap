// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Offcanvas> should match snapshot 1`] = `<DocumentFragment />`;
